fluent-bit-to-loki/pkg/lokiplugin/loki.go

```diff
@@ -98,17 +99,25 @@ func (l *loki) SendRecord(r map[interface{}]interface{}, ts time.Time) error {
 		return fmt.Errorf("error creating line: %v", err)
 	}
 
+	elapsedBeforeSend := time.Since(start)
+	level.Debug(l.logger).Log("SEND PROCESS took ", elapsedBeforeSend.Milliseconds(), "STREAM ", lbs)
```

Comment:
Could you log to which client the logs are sent?
Also, either use `elapsedBeforeSend.String()` or add in the logs message `ms`.